Dockerfile

```diff
@@ -0,0 +1,23 @@
+# An image of ducktape that can be used to setup a Docker cluster where ducktape is run inside the container.
+
+FROM ubuntu:14.04
```

Comment:
Cant we use a light weight OS like alpine? Ubuntu is heavy.

Reply:
That is what we use in muckrake, so we have the use the same, otherwise this image wouldn't be useful at all. Downstream projects can use various OS and maybe we will publish different OSes later if the need comes up, but for now, just Ubuntu.